src/data/items.txt

```diff
@@ -11327,6 +11327,6 @@
 11299
 11300	sleeping patriotic eagle	916117380	pateagle_sleep.gif	grow	t	0
 11301	claw-held flag	498722729	flag2.gif	familiar	t,d	10
-11302
+11302	souvenir flag	557635312	flag1.gif	usable	t,d	10	souvenir flag
```

Comment:
What I meant is, KoL gives you a "use multiple" option. We denote that by saying "multiple", not "usable".
```
> closet put 2 souvenir flag

Placing items into closet...
Requests complete.

> closet take 2 souvenir flag

Removing items from closet...
souvenir flag is multiusable, but KoLmafia thought it was not
You acquire souvenir flag (2)
Requests complete.
```
Additionally:
```
> /closet 10 souvenir flag

You don't have that many souvenir flags, you've only got 2.
```
So the plural actually is "souvenir flags". SInce that's just an added "s", you don't need to list the plural in items.txt